services/core-web/src/styles/settings/theme.scss

```diff
@@ -15,8 +15,9 @@ $ok-green: #45a766;
 $primary-color: $violet;
 $menu-item-color: $white;
 
-$error-color: $alert-red;
-$success-color: $ok-green;
+// $error-color: $alert-red;
```

Comment:
Curious why these are commented out? If they won't be needed, please remove them entirely.

Reply:
Discussed in person.